File: Programming Basics/Conditional Statements Advanced - Lab/07.WorkingHours.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int hour = int.Parse(Console.ReadLine());
        string day = Console.ReadLine();

        switch (day)
        {
            case "Monday":
                if (hour >= 10 && hour <= 18)
                {
                    Console.WriteLine("open");
                }
                else
                {
                    Console.WriteLine("closed");
                }
                break;
            case "Tuesday":
                if (hour >= 10 && hour <= 18)
                {
                    Console.WriteLine("open");
                }
                else
                {
                    Console.WriteLine("closed");
                }
                break;
            case "Wednesday":
                if (hour >= 10 && hour <= 18)
                {
                    Console.WriteLine("open");
                }
                else
                {
                    Console.WriteLine("closed");
                }
                break;
            case "Thursday":
                if (hour >= 10 && hour <= 18)
                {
                    Console.WriteLine("open");
                }
                else
                {
                    Console.WriteLine("closed");
                }
                break;
            case "Friday":
                if (hour >= 10 && hour <= 18)
                {
                    Console.WriteLine("open");
                }
                else
                {
                    Console.WriteLine("closed");
                }
                break;
            case "Saturday":
                if (hour >= 10 && hour <= 18)
                {
                    Console.WriteLine("open");
                }
                else
                {
                    Console.WriteLine("closed");
                }
                break;
            default:
                Console.WriteLine("closed");
                break;
        }
    }
}

File: Programming Basics - Python/Conditional Statements Advanced - Lab/07.WorkingHours.py
hour = int(input())
day = input()

state = ''

if 10 <= hour <= 18:
    if day == 'Monday':
        state = 'open'
    elif day == 'Tuesday':
        state = 'open'
    elif day == 'Wednesday':
        state = 'open'
    elif day == 'Thursday':
        state = 'open'
    elif day == 'Friday':
        state = 'open'
    elif day == 'Saturday':
        state = 'open'
    else:
        state = 'closed'
else:
    state = 'closed'

print(state)